client/me/purchases/product-link/index.jsx

```diff
@@ -42,8 +42,8 @@ const ProductLink = ( { productUrl, purchase, selectedSite } ) => {
 	}
 
 	if ( isGoogleApps( purchase ) ) {
-		url = getGSuiteSettingsUrl( purchase.meta );
-		text = i18n.translate( 'G Suite Settings' );
+		url = emailManagement( selectedSite.slug, purchase.meta );
+		text = i18n.translate( 'Email Settings' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **53** times:
`translate( 'Email settings' )` *ES Score: 8*